commands/init.js

```diff
@@ -130,6 +132,12 @@ module.exports = new Command("init [feature]")
               "Press Space to select features, then Enter to confirm your choices.",
             choices: prompt.convertLabeledListChoices(choices),
           },
+          {
```

Comment:
Alternatively, we can put the gitignore prompt first, which will remove the need of the `are you ready to proceed` hack on windows (see lines above). I'm not quite sure if that's a good idea though. Let me know if you'd prefer it that way.